src/test/java/ru/hogwarts/school/controller/StudentControllerTests.java

```diff
@@ -0,0 +1,208 @@
+        Student john = new Student(-1, "John", 37);
+        studentRepository.save(bob);
+        studentRepository.save(john);
+        Assertions.assertThat(this.restTemplate
+                        .getForObject("/student?age=37", Collection.class).size())
+                .isEqualTo(2);
+    }
+
+    @Test
+    void testDeleteStudent() {
+        Student student = new Student(100, "John", 77);
+        studentRepository.save(student);
+        student = this.restTemplate
+                .postForObject("/student", student, Student.class);
+        restTemplate.delete("/student" + "/" + student.getId());
+        Assertions.assertThat(this.restTemplate
+                .getForObject("/student" + "/" + student.getId(), Student.class))
+                .isEqualTo(new Student(0, null, 0));
+    }
+
+    @Test
+    void testFindStudentByAgeBetween() {
+        Student bob = new Student(-1, "Bob", 18);
+        Student john = new Student(-1, "John", 19);
+        studentRepository.save(bob);
+        studentRepository.save(john);
+        int size = restTemplate
+                .getForObject("/student/findByAgeBetween?min=10&max=30", Collection.class)
+                .size();
+        Assertions.assertThat(size).isEqualTo(2);
+    }
+
+    @Test
+    void testGetFacultyOfStudent() {
+        Faculty faculty = new Faculty();
+        faculty.setName("Математика");
+        faculty.setColor("Синий");
+        Student bob = new Student(-1, "Bob", 18);
+        bob.setFaculty(faculty);
+        facultyRepository.save(faculty);
+        studentRepository.save(bob);
+        Assertions.assertThat(restTemplate
+                .getForObject("http://localhost:" + port + "/student/getFacultyOfStudent/" + bob.getId(), Faculty.class))
+                .isNotNull();
+    }
+
+    @Test
+    void testGetStudentsOfFaculty() {
+        Faculty faculty = new Faculty();
+        faculty.setName("Математика");
+        faculty.setColor("Синий");
+        Student bob = new Student(-1, "Bob", 18);
+        bob.setFaculty(faculty);
+        Student john = new Student(-1, "John", 21);
+        john.setFaculty(faculty);
+        facultyRepository.save(faculty);
+        studentRepository.save(bob);
+        studentRepository.save(john);
+        int size = restTemplate
+                .getForObject("http://localhost:"
+                        + port
+                        + "/student//studentsOfFaculty/" + faculty.getId(), Collection.class)
+                .size();
+        Assertions.assertThat(size).isEqualTo(2);
+    }
+
+    @Test
+    public void testUploadAvatar() throws IOException {
+        Student bob = new Student(-1, "Bob", 34);
+        studentRepository.save(bob);
+        byte[] avatar = Files.readAllBytes(Paths.get("src/main/resources/images/1.jpg"));
+        LinkedMultiValueMap<String, Object> body = new LinkedMultiValueMap<>();
+        body.put("avatar", Collections.singletonList(new ByteArrayResource(avatar)));
```

Comment:
```suggestion
        body.add("avatar", new ClassPathResource("1.jpg"));
```
замени на такую строку